testsuite/features/build_validation/add_activation_keys/add_activation_key.template

```diff
@@ -8,4 +8,6 @@ Feature: Create an activation key for <client>
   I want to use activation keys
 
   Scenario: Create an activation key with the channel and child channels for a <client>
-    When I create an activation key including custom channels for "<client>" via XML-RPC
+    When I am logged in API as user "admin" and password "admin"
+    And I create an activation key including custom channels for "<client>" via API
+    And I logout from API
```

Comment:
I suggest we hide login logout from the API.
And let only `I create an activation key including custom channels for "<client>" via API`, which inside will handle API connection. Those are too technical details that a user don't need to read in the scenario steps.

Reply:
Because for every feature we are starting and stopping a chrome browser process, which is clean and doesn't have cookies of other instances. That impedes us to re-use the login between features. And at same time, it's easier to follow the state of the browser page.
And API is a bit different, we just send a request and have a response, so it's somehow stateless in that sense.

Reply:
Makes sense.

OK, let's wait for Can's response.

The fact that the old code was logging in and out all the time makes me think it's the same as the UI, tokens have a limited lifespan, and probably less than 3 hours.

Reply:
Can's answer:

Eric Bischoff  19 h 04
```
do you know how long the token remains valid ?
```

Can Bulut Bayburt  19 h 04
```
same as the web UI
it's the same token
```

so I suggest to re-login at a feature level, as we do for the UI.

Reply:
Actually, most of the features I'm reviewing have login/logut by scenario, multiple times in the same feature.
It doesn't seems the exception, maybe we should take another look and see if we can remove some of them.

Reply:
We definitely should. But in another PR.